remoteworkitem/trackeritem_repository.go

```diff
@@ -8,3 +8,18 @@ func upload(db *gorm.DB, tqID int, item map[string]string) error {
 	err := db.Create(&ti).Error
 	return err
 }
+
+func convert(db *gorm.DB, tqID int, item map[string]string, provider string) error {
```

Comment:
@sbose78 Go ahead and introduce a separate TX for each tracker item and corresponding work items (inside the for loop)